popSimManu.tex

```diff
@@ -220,9 +220,9 @@ \section*{Introduction}
 realistic simulations for population genetic analysis a simple matter of
 choosing pre-implemented models from a community-maintained catalog.
 The \stdpopsim catalog currently contains six species: humans,
-\textit{Pongo abelii},
+\textit{Pongo abelii}, \textit{Canis familiaris},
```

Comment:
this ordering of organisms in the manuscript was driving me nuts. see https://github.com/popsim-consortium/stdpopsim/issues/531